src/__experimental__/components/numeral-date/numeral-date.d.ts

```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
+import { MarginSpacingProps } from '../../../utils/helpers/options-helper';
 
-interface DayMonthDate {
+interface DayMonthDate  extends MarginSpacingProps {
```

Comment:
Wrong interface, this should be done on line 26th in this file

Reply:
My bad :D changed, thanks